Python/Video to PDF(CLI)/README.md

```diff
@@ -0,0 +1,13 @@
+# Video to PDF
+Convert any video of your choice into a pdf file!
+
+## How to use this script?
+- Enter the file path of the video from your local machine.
+- The script will convert it into a pdf and save it as `my_pdf.pdf` in the same location where the video was present.
+
+To install all the dependencies just run the command
+
+``
+pip install -r requirements.txt
+``
```

Comment:
Add a usage instruction

Reply:
Kindly resolve this by adding a usage instruction.

Reply:
Add a ss of terminal where script is running.